test/integration/script-loader/pages/page4.js

```diff
@@ -0,0 +1,38 @@
+import Script from 'next/experimental-script'
+
+const url =
+  'https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.20/lodash.min.js'
+
+const Page = () => {
+  return (
+    <div class="container">
+      <Script
+        src={url}
+        key="script1"
+        onLoad={() => {
+          // eslint-disable-next-line no-undef
+          document.getElementById('text').textContent += _.repeat('a', 3)
+        }}
```

Comment:
I have a feeling allowing `onLoad` to be an inline function will bite us in suboptimal ways. Users may end up importing a large library in the top of module scope. Maybe we could force `onLoad` and `onError` handlers to be dynamic imports? WDYT @timneutkens?

Reply:
How would users import large libs in onload? By injecting script tags?

Reply:
What @Timer seems to be referring to is that the `onLoad` function could be `import()`ed causing the code to be only loaded when the JS is loaded and not always included in the bundle. However we can investigate that further later